File: 2st/Yeonwoo/배열 만들기 2.py
def solution(l, r):
    answer = []
    for i in range(l,r+1):
        if str(i).count('0') + str(i).count('5') == len(str(i)):
            answer.append(i)
    if answer == []:
        answer.append(-1)
    return answer

File: 4st/Gau/programmer/배열 만들기 2.java
import java.util.*;

class Solution {
    public List solution(int l, int r) {
        List<Integer> answer = new ArrayList<>();
        
        for (int i = l; i <= r; i++) {
            String s = i + "";
            boolean bool = true;
            for (char c : s.toCharArray()) {
                if (c != '0' && c != '5') {
                    bool = false;
                }
            }
            if (bool) {
                answer.add(i);
            }
        }
        if (answer.isEmpty()) {
            answer.add(-1);
        }
        
        return answer;
    }
}